.github/workflows/build.yaml

```
@@ -90,3 +99,26 @@ jobs:
       run: |
         cargo install --path .
         agora --version
+
+    - name: Package
+      id: package
+      if: startsWith(github.ref, 'refs/tags/')
+      env:
+        TARGET: ${{ matrix.target }}
+        REF: ${{ github.ref }}
+        OS: ${{ matrix.os }}
+      run: ./bin/package
+
+    - name: Pre-release Check
```

```suggestion
    - name: Prerelease Check
```

Since that's how we spell it elsewhere.